highfidelity/hifi
# 7587 Lasers resize with avatar

This PR creates the option for making the lasers scale with the avatar. 
  
https://highfidelity.fogbugz.com/f/cases/7587
...

## TEST PLAN
- Change the size of your avatar and make it big.
- Use the lasers. They should look scaled accordingly. (no ultra thin lasers)
- Change the size of your avatar and make it small.
- Use the lasers. They should look scaled accordingly. (no ultra thick lasers)
- Close HF and edit the file \scripts\system\controllers\controllerModules\FarActionGrabEntity.js
- Add a **lineWidth** parameter to the **fullPath** object `lineWidth: 1.1,`
- Save and restart HF. Now the trigger should project a thick laser. (The tablet should be closed)
- Make sure that the thick laser scale with the avatar.
- Look for the **scaleWithAvatar** parameter in the file _\scripts\system\controllers\controllerModules\FarActionGrabEntity.js_ and change it to false, or erase it.
- Save and restart HF. Now the lasers should not scale with the avatar.
